javascripts/discourse-linkify/lib/utilities.js.es6

```diff
@@ -126,19 +126,33 @@ const modifyText = function(text, action) {
   }
 }
 
-const modifyNode = function(elem, action, skipTags) {
+const isSkippedClass = function(classes, skipClasses) {
+
+  if (!classes) return false;
+
+  // Return true if at least one of the classes should be skipped
+  return classes.split(" ").some(cls => {
+    for (let skipClass in skipClasses) {
+      if (cls === skipClass) return true;
+    }
+    return false;
+  });
```

Comment:
Isn't `skipClasses` an hash? You should be able to do much shorter 😉 

```suggestion
  return classes.split(" ").some(cls => cls in skipClasses);
```

Reply:
Ah, of course. :man_facepalming: Thanks! This also exposed a silly bug that I just fixed.